tests/src/Tests/ShaderUtils.cpp

```diff
@@ -353,7 +353,7 @@ void ExpectNZSL(const nzsl::Ast::Module& shaderModule, std::string_view expected
 	}
 }
 
-void ExpectSPIRV(const nzsl::Ast::Module& shaderModule, std::string_view expectedOutput, const nzsl::ShaderWriter::States& options, const nzsl::SpirvWriter::Environment& env, bool outputParameter)
+void ExpectSPIRV(const nzsl::Ast::Module& shaderModule, std::string_view expectedOutput, const nzsl::ShaderWriter::States& options, const nzsl::SpirvWriter::Environment& env, bool outputParameter, bool validateScalarLayout)
```

Comment:
It would be more future-proof to add a `const spvtools::ValidatorOptions& validatorOptions` parameter with a default value?